packages/sourcecred/config/webpack.config.web.js

```diff
@@ -181,9 +181,11 @@ function makeConfig(mode /*: "production" | "development" */) /*: Object */ {
     module: {
       strictExportPresence: true,
       rules: [
-        // TODO: Disable require.ensure as it's not a standard language feature.
-        // We are waiting for https://github.com/facebookincubator/create-react-app/issues/2176.
-        // { parser: { requireEnsure: false } },
+        {
+          parser: {
+            requireEnsure: false,
+          },
+        },
```

Comment:
I appreciate you catching this, but it'd be cleaner to make this its own PR

Reply:
That makes sense, I fixed that